src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig

```diff
@@ -0,0 +1,184 @@
+        </div>
+        <div class="col-8">
+          {{ customerInformation.personalInformation.shopName }}
+        </div>
+      </div>
+    {% endif %}
+
+    <div class="row mb-1">
+      <div class="col-4 text-right">
+        {{ 'Language'|trans({}, 'Admin.Global') }}
+      </div>
+      <div class="col-8">
+        {{ customerInformation.personalInformation.languageName }}
+      </div>
+    </div>
+
+    <div class="row mb-1">
+      <div class="col-4 text-right">
+        {{ 'Registrations'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </div>
+      <div class="col-8">
+        {% set isNewsletterSubscribed = customerInformation.personalInformation.subscriptions.newsletterSubscribed %}
+        {% set isPartnerOffersSubscribed = customerInformation.personalInformation.subscriptions.partnerOffersSubscribed %}
+
+        <span class="badge badge-{% if isNewsletterSubscribed %}success{% else %}danger{% endif %} rounded pt-0 pb-0">
+          <i class="material-icons">{% if isNewsletterSubscribed %}check{% else %}cancel{% endif %}</i>
+          {{ 'Newsletter'|trans({}, 'Admin.Global') }}
+        </span>
+
+        <span class="badge badge-{% if isPartnerOffersSubscribed %}success{% else %}danger{% endif %} rounded pt-0 pb-0">
+          <i class="material-icons">{% if isPartnerOffersSubscribed %}check{% else %}cancel{% endif %}</i>
+          {{ 'Partner offers'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        </span>
+      </div>
+    </div>
+
+    <div class="row mb-1">
+      <div class="col-4 text-right">
+        {{ 'Latest Update'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </div>
+      <div class="col-8">
+        {{ customerInformation.personalInformation.lastUpdateDate }}
+      </div>
+    </div>
+
+    <div class="row mb-1">
+      <div class="col-4 text-right">
+        {{ 'Status'|trans({}, 'Admin.Global') }}
+      </div>
+      <div class="col-8">
+        {% set isCustomerActive = customerInformation.personalInformation.active %}
+
+        <span class="badge badge-{% if isCustomerActive %}success{% else %}danger{% endif %} rounded pt-0 pb-0">
+          {% if isCustomerActive %}
+            <i class="material-icons">check</i>
+            {{ 'Active'|trans({}, 'Admin.Global') }}
+          {% else %}
+            <i class="material-icons">cancel</i>
+            {{ 'Inactive'|trans({}, 'Admin.Global') }}
+          {% endif %}
+        </span>
+      </div>
+    </div>
+
+    {% if customerInformation.personalInformation.guest %}
+      <div class="row mb-1">
+        <div class="col-4 text-right">
+          {{ 'This customer is registered as a Guest.'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        </div>
+        <div class="col-8">
+          {% if customerInformation.generalInformation.customerBySameEmailExists %}
+            <p>{{ 'A registered customer account using the defined email address already exists. '|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

Comment:
Watch the space at the end of the wording, unless it is made on purpose?

Reply:
translation key contains this space. :/